pkg/provider/azure_loadbalancer_backendpool_test.go

```diff
@@ -247,10 +238,77 @@ func TestEnsureHostsInPoolNodeIP(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc:  "local service",
```

Comment:
could you make the desc for those tests more clear? e.g. something like local service should xxxx.